apps/explorer/lib/hosts.ts

```diff
@@ -75,3 +77,27 @@ export function rankHosts(hosts: ExplorerHost[] | undefined) {
     }))
     .sort((a, b) => b.score - a.score) // Sort descending
 }
+
+export async function getCachedTopHostsFunc() {
```

Comment:
Since this method does not include caching logic you could name it `getTopHosts`.